requirements/dev.in

```diff
@@ -4,9 +4,6 @@
 #    pip-compile --output-file requirements/dev.txt requirements/dev.in
 #
 -e git+https://github.com/kytos-ng/python-openflow.git#egg=python-openflow
--e git+https://github.com/kytos-ng/of_core.git#egg=kytos-of_core # coloring dependency
--e git+https://github.com/kytos-ng/kytos.git#egg=kytos
--e git+https://github.com/kytos-ng/coloring.git#egg=amlight-coloring
--e .[dev]
-pytest-asyncio==0.18.3
-black==22.3.0
+-e git+https://github.com/kytos-ng/of_core.git#egg=kytos-of_core
+-e git+https://github.com/kytos-ng/kytos.git@feat/starlette#egg=kytos[dev]
```

Comment:
Once `kytos-ng/kytos` `feat/starlette` branch lands, I'll revert this branch here.